cc/main.cc

```diff
@@ -16,6 +16,7 @@
 #include <unistd.h>
 #include <cstring>
 #include <functional>
+#include <iomanip>
```

Comment:
please use absl::StreamFormat for consistency with the rest of the codebase

Reply:
Done.